README.md

```diff
@@ -1,9 +1,22 @@
 # Akamai EdgeWorkers Examples
 
+
 Akamai EdgeWorkers helps deliver superior web experiences by enabling developers to run JavaScript at the Edge to customize web traffic, rather than using origin servers for these tasks. EdgeKV provides a global, low-latency key-value data store at the Edge to complement EdgeWorkers. The code snippets in this repository represent a starting point to help you build logic that will solve a range of use-cases via Akamai EdgeWorkers and EdgeKV.
 
 These code samples are offered as-is, and are working code samples within the EdgeWorkers and EdgeKV products.  Feel free to use, modify and extend what you find here!
 
+## Contributing 
+
+When contributing to the repository please describe the change or examples in detail in the pull request.
+
+### contributing examples
+- create a pull request against the `publicexamples` folder.
+- Once the pull request is created, If the contributing user has not previously signed a Contributor License Agreement (CLA), they must complete the CLA signing steps as indicated in the pull request. The CLI signature will be stored in a separate and private repository named cla-sign.
```

Comment:
"The CLI Signature" --> "The CLA signature"

Reply:
Because cla-sign is a private repository, it may not be worth mentioning in the readme. Maybe we can just say that "The CLI signature will be stored in an Akamai private repository in Github."